src/main/groovy/com/morpheusdata/commvault/backup/CommvaultBackupExecutionProvider.groovy

```diff
@@ -312,6 +355,10 @@ class CommvaultBackupExecutionProvider implements BackupExecutionProvider {
 						return rtn
 					}
 				}
+				// disable cloud init and clear cache to force cloud init on restore
```

Comment:
Changes in executeBackup aren't related to these stories and will cause merge conflicts.

Reply:
@ddeyoung removed the executeBackup changes as suggested, as this changes is not part of these stories.